src/ui/Roster.ts

```diff
@@ -270,14 +377,26 @@ export default class Roster {
    }
 
    private insert(rosterItem: RosterItem) {
-      let contactList = this.contactList;
       let contact = rosterItem.getContact();
+      let contactType = contact.getType();
+      let list = (contactType==='chat'?this.contactList:this.groupList);
       let contactName = contact.getName();
 
-      let lastMessageDate = contact.getLastMessageDate();
+      if (!$($(list).parent()[0]).prev().hasClass('jsxc-active'))
```

Comment:
This is really hard to read.

Reply:
now using this:

    let parentElement = $(list).parent();
      let previousParent = $(parentElement).prev();

      if (!previousParent.hasClass('jsxc-active'))
      {
         parentElement.css('max-height','');
         parentElement.addClass('jsxc-roster-item-filtered');
      }
      else
          if ($($(list).parent()[0]).prev().hasClass('jsxc-active'))
          {
              parentElement.removeClass('jsxc-roster-item-filtered');
              parentElement.css('max-height','100%');
          }